geomstats/geometry/special_euclidean.py

```diff
@@ -115,12 +122,32 @@ def belongs(self, point, point_type=None):
             point = gs.to_ndarray(point, to_ndim=2)
             n_points, point_dim = point.shape
             belongs = point_dim == self.dimension
-            belongs = gs.to_ndarray(belongs, to_ndim=1)
-            belongs = gs.to_ndarray(belongs, to_ndim=2, axis=1)
-            belongs = gs.tile(belongs, (n_points, 1))
+            belongs = gs.logical_and(belongs,
+                                     self.rotations.belongs(point[:, :self.n]))
+            # TODO: Should to_ndarray do this if to_ndim = ndim(x) - 1 ?
```

Comment:
TODO syntax needs a github account name, eg: TODO(pchauchat).

This way, you can search for all your TODOs in the codebase easily.

Reply:
Also, NIT: TODO items should be actionable, eg:
TODO(pchauchat): Decide if to_ndarray.... .

Reply:
I didn't know who to assign this to. Since I'm new here, and it involves a quite important backend function, I mainly wanted to raise the issue.

Reply:
Makes sense! In this case:
- either raise a issue and reference the line (go in the code, click on the number of line, then `Copy permalink` and paste this into the issue post), - preferred.
- or assign me x) TODO(ninamiolane) -> use my github login.

Reply:
@pchauchat update on this?